features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/ResourcesLRUCacheTest.kt

```diff
@@ -38,32 +38,23 @@ import org.mockito.quality.Strictness
 )
 @MockitoSettings(strictness = Strictness.LENIENT)
 @ForgeConfiguration(ForgeConfigurator::class)
-internal class Base64LRUCacheTest {
-    private lateinit var testedCache: Base64LRUCache
+internal class ResourcesLRUCacheTest {
+    private lateinit var testedCache: ResourcesLRUCache
 
-    private lateinit var internalCache: LruCache<String, CacheData>
+    private lateinit var internalCache: LruCache<String, ByteArray>
 
     @Mock
     lateinit var mockDrawable: Drawable
 
     @Mock
     lateinit var mockInvocationUtils: InvocationUtils
 
-    @StringForgery
-    lateinit var fakeBase64Encoding: String
-
-    private lateinit var fakeCacheData: CacheData
-
     val argumentCaptor = argumentCaptor<String>()
 
     @BeforeEach
-    fun setup(forge: Forge) {
-        val fakeResourceId = forge.aNullable { aString() }
-        val fakeBase64ByteArray = fakeBase64Encoding.toByteArray(Charsets.UTF_8)
-        val fakeResourceIdByteArray = fakeResourceId?.toByteArray(Charsets.UTF_8)
-        fakeCacheData = CacheData(fakeBase64ByteArray, fakeResourceIdByteArray)
-        internalCache = LruCache<String, CacheData>(MAX_CACHE_MEMORY_SIZE_BYTES)
-        testedCache = Base64LRUCache(
+    fun setup() {
+        internalCache = LruCache<String, ByteArray>(MAX_CACHE_MEMORY_SIZE_BYTES)
```

Comment:
it seems this one can be simply a local variable rather than a property, it is used only in `setup` method